pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -209,6 +210,12 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 		}
 	}
 
+	// fixes https://issues.redhat.com/browse/OCPBUGS-5304
+	if reflect.DeepEqual(liveVotingMembers, healthyLiveVotingMembers) && len(healthyLiveVotingMembers) <= desiredControlPlaneReplicasCount {
```

Comment:
This whole function could really benefit from restructuring to make this more readable but I'm okay with keeping this change small to unblock us on fixing the scaling CI.

We can do a follow up PR to clean up this controller.